DrawerController/DrawerController.swift

```diff
@@ -194,6 +199,7 @@ private class DrawerCenterContainerView: UIView {
 }
 
 open class DrawerController: UIViewController, UIGestureRecognizerDelegate {
+  public var delegate: DrawerControllerDelegate?
```

Comment:
We should change this to `weak` and add :class to the protocol, otherwise it's easy to cause retain cycle